### DISCUSSION TITLE
Is it support MudBlazor component?

### DISCUSSION THREAD

**Discussion by psangrassamee** [jsakamoto/BlazingStory#15] [Q&A] [answered]
I try to add MudBlazor component but it render button without style
![image](https://github.com/jsakamoto/BlazingStory/assets/127803845/d5300905-877e-4cb2-8a0e-99c10f1756c7)

**Comment by KrzychuR** [comment, marked as answer]
@psangrassamee I'm using it without any issue with MudBlazor, even with combination with Telerik components (!) - so make sure that  `DefaultLayout.razor` is configured properly and add proper CSS/JS links to `iframe.html`

**Reply by psangrassamee** [reply]
Thank you I misconfigure some about MudBlazor.
